india_compliance/gst_india/doctype/bill_of_entry/bill_of_entry.py

```
@@ -791,7 +791,14 @@ def get_pi_items(purchase_invoices):
 
 
 @frappe.whitelist()
-def fetch_pending_boe_invoices(doctype, txt, searchfield, start, page_len, filters):
+def fetch_pending_boe_invoices(
+    doctype: str,
```

Ruff (0.14.14)

[warning] Unused function argument: `doctype`

(ARG001)